test/test_site.rb

```diff
@@ -192,12 +192,14 @@ def generate(site)
         index.html
         index.html
         main.scss
-        main.scss
         properties.html
         sitemap.xml
         static_files.html
-        symlinked-file
       )
+      unless Jekyll::Utils::Platforms.windows?
```

Comment:
Because as mentioned before, those kind of couplings are terrifying, and they're gonna make our eventual move to RSpec harder.  If you need a conditional test use contexts:

```
context "when not on Windows"
   it "should do this stuff"
     skip_if_windows

context "on Windows"
   it "should do stuff"
```

Sorry I didn't elaborate that.